### ISSUE TITLE
Use generative AI to generate explanation of elements

### ISSUE THREAD

**Issue by Taliesin1** [Azgaar/Armoria#154]
As you know, charges have symbology behind them, as do the colors, patterns, and even lines. It would be awesome to have a curated dictionary of these and use generative AI to provide an explanation of the arms, and what they traditionally symbolize.

**Comment by Azgaar** [comment]
In heraldry don't have some defined meaning, only the COA author knows it's meaning and COA is always open for interpretation. So I would rather not do it and allow users to define meanings themself.

**Comment by Taliesin1** [comment]
Fair enough, but one could always say, "These are the traditional meaning ascribed to these heraldic elements, which are provided for ideation and context only. Hopefully, it will inspire you to assign your own meanings to the arms generated with Armorial."
